packages/core/src/types.ts

```diff
@@ -80,6 +81,7 @@ export interface IPrismInput<I> {
   data: I;
   validations: {
     input: IPrismDiagnostic[];
+    security: AuthErr[];
```

Comment:
Hey @lag-of-death, curious about that, was `validations.input` not possible/easy to reuse?

Reply:
@chris-miaskowski, I don't know. I didn't want to reuse input. If you'd see `security` to be part of `input` I think it won't be hard to put it there.